python/py-httplib2/Portfile

```diff
@@ -4,8 +4,8 @@ PortSystem        1.0
 PortGroup         python 1.0
 
 name              py-httplib2
-set realversion   0.20.4
-revision          1
+set realversion   0.21.0
```

Comment:
does this new version still support everything down to Python 2.7?

Reply:
it seems like it https://github.com/httplib2/httplib2/blob/master/setup.py

Reply:
great - to be honest I didn't check, was just reviewing the PR on my phone which make going to upstream websites a bit cumbersome ;)